src/content/structured/components/pagination/guidance.mdx

```diff
@@ -41,6 +43,12 @@ Pagination enables large amounts of content to be split across multiple pages so
 
 ## Component variants
 
+<IcAlert
+  heading="Coming soon"
+  message="The carousel functionality is in development."
```

Comment:
Should this be:
```suggestion
  message="The carousel pagination variant is in development."
```
based on the description in the ticket? https://github.com/mi6/ic-design-system/issues/464